### ISSUE TITLE
Spaces removed incorrectly (tag-based code)

### ISSUE THREAD

**Issue by djiest** [jcberquist/commandbox-cfformat#27]
I'm working on a Application that has the following code in it
```
<cfif len(event.getValue('filter_service','')) eq 0
        AND len(event.getValue('filter_client','')) eq 0
        AND len(event.getValue('filter_filenumber','')) eq 0
        AND len(event.getValue('filter_selfrom','')) eq 0
        AND len(event.getValue('filter_selto','')) eq 0
        AND len(event.getValue('filter_period','')) eq 0
        AND len(event.getValue('filter_status','')) eq 0
        AND len(event.getValue('q','')) eq 0
        and not isDefined("rc.medicalReportsNeedApproval")>
```

When I run `cfformat` on it, with the default settings, it removes the space between `AND` and `len(event.getValue....`

```
<cfif len(event.getValue('filter_service', '')) eq 0
        ANDlen(event.getValue('filter_client', '')) eq 0
        ANDlen(event.getValue('filter_filenumber', '')) eq 0
        ANDlen(event.getValue('filter_selfrom', '')) eq 0
        ANDlen(event.getValue('filter_selto', '')) eq 0
        ANDlen(event.getValue('filter_period', '')) eq 0
        ANDlen(event.getValue('filter_status', '')) eq 0
        ANDlen(event.getValue('q', '')) eq 0
        andnot isDefined('rc.medicalReportsNeedApproval')>
```
when I first refactor it to cfscript, and then run cfformat, this issue does NOT occur

**Comment by jcberquist** [comment]
Thats not good :). I tried creating a bare bones cfc like so:
```cfc
<cfcomponent>
<cfif len(event.getValue('filter_service','')) eq 0
    AND len(event.getValue('filter_client','')) eq 0
    AND len(event.getValue('filter_filenumber','')) eq 0
    AND len(event.getValue('filter_selfrom','')) eq 0
    AND len(event.getValue('filter_selto','')) eq 0
    AND len(event.getValue('filter_period','')) eq 0
    AND len(event.getValue('filter_status','')) eq 0
    AND len(event.getValue('q','')) eq 0
    and not isDefined("rc.medicalReportsNeedApproval")>
</cfif>
</cfcomponent>
```
I ran `cfformat` on that with the default settings and it did not remove the space. Is there some more context around that `<cfif>`? If you run the command on my example above, does it remove the spaces?

**Comment by djiest** [comment]
Hi,
see attached file. With this file I can recreate the issue.
[foo.zip](https://github.com/jcberquist/commandbox-cfformat/files/3313362/foo.zip)

I'm working with 
commandbox v4.7.0+00026
cfformat v0.8.0

on Windows10 64-bit

**Comment by jcberquist** [comment]
This was perfect, thanks! It turns out to be a bug in the Sublime Text CFML syntax I use - `AND` is not matched correctly after a new line. I have fixed the syntax upstream, and this will be fixed with the next release here.

**Comment by djiest** [comment]
Great, I'll await the update

**Comment by jcberquist** [comment]
This should be fixed in v0.9.0

**Comment by djiest** [comment]
Hi John

When will version v0.9.0 be available on Forgebox? Since that's where
Commandbox gets the updates from.

Regards

Joost


Op zo 23 jun. 2019 om 23:07 schreef John Berquist <notifications@github.com
